app/_locales/en/messages.json

```diff
@@ -430,6 +430,9 @@
   "chainIdExistsErrorMsg": {
     "message": "This Chain ID is currently used by the $1 network."
   },
+  "chooseAccounts": {
+    "message": "Choose the account(s) to use on this site"
+  },
```

Comment:
Although this message matches the designs, I think the verb should be "select" here, not "choose". We overwhelmingly use "select" over "choose" in our existing locale messages (57 vs 2 instances), and we should do it for consistency if no other reason.

The remaining changes to the message are fine. We can probably rename the key to "selectAccounts" and keep the translations if we make this change. We just need to highlight the change to our translators next week.